apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -146,6 +148,7 @@ class GameLabVisualizationColumn extends React.Component {
 
     return (
       <span>
```

Comment:
nit: I know this PR isn't the one that added this element, but this should _also_ be a div rather than a span

Reply:
Due to the time crunch on this change. I won't fix this now.